lib/generators/statesman/templates/create_migration.rb.erb

```diff
@@ -9,6 +9,8 @@ class Create<%= migration_class_name %> < ActiveRecord::Migration
       t.timestamps null: false
     end
 
+    add_foreign_key :<%= table_name %>, :<%= parent_table_name %>
```

Comment:
I deliberately kept it out of there, under the assumption that the end user would have already added a foreign key to the existing table they're updating if they wanted one.